brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageMergingRollbackHandlerTest.java

```diff
@@ -33,6 +32,7 @@
 import no.unit.nva.identifiers.SortableIdentifier;
 import no.unit.nva.model.Publication;
 import no.unit.nva.model.testing.PublicationGenerator;
+import no.unit.nva.publication.exception.GatewayTimeoutException;
```

Comment:
Only added GatewayTimeoutException to method signatures and removed unused imports.